pkg/config/loader.go

```
@@ -0,0 +1,817 @@
+	foundPaths, err = cl.SearchAtmosConfigFileDir(searchDir)
+	if err != nil {
+		log.Debug("Failed to find git atmos config file in path", "path", searchDir, "error", err)
+	}
+	if len(foundPaths) == 0 {
+		log.Debug("Failed to find config file in git root", "path", searchDir)
+	} else {
+		atmosFoundFilePaths = append(atmosFoundFilePaths, foundPaths...)
+	}
+
+	if len(atmosFoundFilePaths) == 0 {
+		return nil, fmt.Errorf("Failed to find config files in git root path", "path", gitRootDir)
```

golangci-lint (1.62.2)

printf: fmt.Errorf call has arguments but no formatting directives

(govet)